### PR TITLE
feat: providing functionality to use else template in if feature directive

### PR DESCRIPTION
**Pull request by itssharmasandeep**
## Description
After this, will be able to provide else template for `*htIfFeature`, similar to `*ngIf`
Example
```
<ng-container *htIfFeature="state as featureState; else elseTemplate">
  <div class="test-div">{{featureState}}</div>
</ng-container>
<ng-template #elseTemplate>
  <div class="else-div">Else template</div>
</ng-template>
```

### Testing
Manual testing done.

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/components/src/feature-check/if-feature.directive.ts [hunk 1]
```diff
@@ -9,7 +9,11 @@ export class IfFeatureDirective implements OnChanges {
   @Input('htIfFeature')
   public featureState?: FeatureState;
 
-  private embeddedViewRef?: EmbeddedViewRef<FeatureFlagsContext>;
+  // tslint:disable-next-line:no-input-rename
```
Comment (anandtiwary) [inline projects/components/src/feature-check/if-feature.directive.ts:12]: why is this needed?

**Review by anandtiwary — COMMENTED** [review]
LGTM. but let's wait for @aaron-steinfeld to take a look as well

File: projects/components/src/feature-check/if-feature.directive.ts (hunk 1 above)
Comment (itssharmasandeep) [reply projects/components/src/feature-check/if-feature.directive.ts:12]: It gives lint error
```
In the class "IfFeatureDirective", the directive input property "elseContent" should not be renamed.
However, you should use an alias when the directive name is also an input property, and the directive name
doesn't describe the property. In this last case, you can disable this rule with `tslint:disable-next-line:no-input-rename`.
```

**Review by aaron-steinfeld — APPROVED** [review]
